Prepare website for OLS-9 cohort (#790)
open-life-science/open-life-science.github.io: openseeds/ols-9/speaker-guide.md

---
layout: page

> title: OLS Speaker Guide

Reviewer: ```suggestion
title: Open Seeds Speaker Guide
```

image: https://images.unsplash.com/photo-1586985564259-6211deb4c122?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1215&q=80
photos:
  name: Visuals on Unsplash
  license: CC-0
  url: https://unsplash.com/photos/Y4qzW3AsvqI
redirect_from: /ols-speaker-guide
---

Thank you for considering speaking at our upcoming Cohort call! We look forward to welcoming you. **🎉🥳️👏**

> # About OLS

Reviewer: ```suggestion
# About Open Seeds
```

> The [OLS program]({% link openseeds/index.md %}) is

Reviewer: ```suggestion
The [Open Seeds program]({% link openseeds/index.md %}) is
```

designed for the early-stage researchers and potential academic leaders
in becoming Open Science ambassadors in their communities.

This program is led by the OLS team members, i.e. us - Bérénice Batut,

> Malvika Sharan, Yo Yehudi and Emmy Tsang. You can reach out to us by emailing

Reviewer: ```suggestion
Malvika Sharan, Yo Yehudi. You can reach out to us by emailing
```

Reviewer: Who else?

[{{ site.email }}](mailto:{{ site.email }}).

We will be running our [9th cohort]({% link openseeds/ols-9/projects-participants.md %})

> (Sep 2021 - Jan 2022).

Reviewer: To be updated

A complete syllabus of this cohort can be found [here]({% link openseeds/ols-9/schedule.md %}).

# About our Code of Conduct

All our members, participants and speakers are expected to abide by our [Code of conduct & community participation guidelines]({% link code-of-conduct.md %}) that apply to all our community spaces including online calls.

If you experience or witness any unacceptable behaviour, or have any other concerns, please report it by contacting the organisers - Bérénice, Malvika and Yo. ([{{ site.email }}](mailto:{{ site.email }})).

> To report an issue involving one of the organisers, please email one of the members individually (([berenice@we-are-ols.org](mailto:berenice@we-are-ols.org), [malvika@we-are-ols.org](mailto:malvika@we-are-ols.org), [yo@we-are-ols.org](mailto:yo@we-are-ols.org)), [emmy@we-are-ols.org](mailto:emmy@we-are-ols.org)).

Reviewer: To be updated

Author: Done. Thanks!
